src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -118,6 +136,54 @@ private async Task VerifyAdditionalFileFixAsync(string language, string source,
 
         #region Diagnostic tests
 
+        [Fact]
+        public async Task AnalyzerFileMissing_Shipped()
+        {
+            var source = @"
+public class C
+{
+    private C() { }
+}
+";
+
+            string shippedText = null;
+            string unshippedText = @"";
```

Comment:
Can we add a test where both `shippedText` and `unshippedText` are empty files, and verify there is no diagnostic?